pkg/state/s3/impl_test.go

```diff
@@ -23,18 +23,29 @@ import (
 	"github.com/minio/minio-go"
 )
 
+const (
+	endpoint   = "localhost:9000"
+	bucketName = "test"
+)
+
 func TestJsonFileSystem(t *testing.T) {
-	client, err := minio.New("localhost:9000", "KubicornAccess", "KubicornSecret", false)
+	client, err := minio.New(endpoint, "KubicornAccess", "KubicornSecret", false)
 	if err != nil {
 		t.Fatalf("Error creating client: %v", err)
 	}
 
-	err = client.MakeBucket("test", "")
+	// If tests fails, you may need to run locally:
+	// docker run -p 9000:9000 --name minio1   -e "MINIO_ACCESS_KEY=KubicornAccess"   -e "MINIO_SECRET_KEY=KubicornSecret" minio/minio server /data
+	// to get this test to pass. For that reason will be skipping this step if it fails.
+	err = client.MakeBucket(bucketName, "")
 	if err != nil {
-		exists, err := client.BucketExists("test")
-		if err != nil || !exists {
-			t.Fatalf("Cannot create bucket: %#v", err)
-		}
+		t.Skipf("could not create bucket, skipping test, please check if minio server is running: %#v", err)
+
+		// Commenting below out for error message above and skipping tests if locally fails for a dev.
+		//exists, err := client.BucketExists(bucketName)
```

Comment:
nit: maybe i missed something. but do we still need to comment this out if its behind the `t.Skip`?